code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -228,10 +228,10 @@ emp_act
 				informed = 1
 			switch(severity)
 				if(1)
-					L.take_damage(0,10)
+					L.take_damage(0,30)

```

Comment:
so the full 6 augments will deal a massive 180 damage to you whenever some git fires the ion rifle at anything within 2 tiles of you?

Augs aren't /that/ good